src/main/java/domain/pieces/Guard.java

```diff
@@ -0,0 +1,48 @@
+package domain.pieces;
+
+import static domain.pieces.PieceNames.GUARD;
+
+import domain.Team;
+import domain.board.PieceOnRoute;
+import domain.board.Point;
+import execptions.JanggiArgumentException;
+import java.util.List;
+
+public final class Guard implements Piece {
+
+    private final Team team;
+
+    public Guard(final Team team) {
+        this.team = team;
+    }
+
+    @Override
+    public boolean hasEqualTeam(final Team team) {
+        return this.team.equals(team);
+    }
+
+    @Override
+    public boolean isAbleToArrive(final Point startPoint, final Point arrivalPoint) {
+        throw new JanggiArgumentException("신하는 이동할 수 없습니다.");
+    }
+
+    @Override
+    public List<Point> getRoutePoints(final Point startPoint, final Point arrivalPoint) {
+        throw new JanggiArgumentException("신하는 이동할 수 없습니다.");
+    }
+
+    @Override
+    public boolean isMovable(final PieceOnRoute pieceOnRoute) {
+        throw new JanggiArgumentException("장군은 이동할 수 없습니다.");
+    }
```

Comment:
```suggestion
    public boolean isMovable(final PieceOnRoute pieceOnRoute) {
        throw new JanggiArgumentException("신하는 이동할 수 없습니다.");
    }
```
움직일 수 없다면 exception이 아니라 false를 return해야하는게 아닐까요?

Reply:
step1 의 미션이 장군과 신하는 움직이지 않는 것으로 한정 되어 있어서 step2 에 구현하려고 지금은 `General` 과 `Guard` 의 내부 메서드들을 이렇게 처리해뒀어요!